prov/verbs/src/fi_verbs.h

```diff
@@ -268,6 +282,8 @@ struct fi_ibv_cq {
 	ofi_atomic32_t		nevents;
 	struct util_buf_pool	*epe_pool;
 	struct util_buf_pool	*wce_pool;
+	struct util_buf_pool	*wre_pool;
+	struct dlist_entry	wre_list;
```

Comment:
Do we need separate wre_pool per CQ and also for SRQ? Can't we just maintain a single wre_pool per endpoint which can be used for both send and recv?

Reply:
Thanks. I've re-implemented this